projects/schematics/src/shared/lib-configs/asm-schematics-config.ts

```diff
@@ -53,3 +58,43 @@ export const ASM_SCHEMATICS_CONFIG: SchematicConfig = {
   },
   dependencyFeatures: [USER_PROFILE_FEATURE_NAME],
 };
+
+export const CUSTOMER_360_MODULE = 'Customer360Module';
+export const CUSTOMER_360_ROOT_MODULE = 'Customer360RootModule';
+export const CUSTOMER_360_MODULE_NAME = 'Customer360';
+export const CUSTOMER_360_FEATURE_NAME_CONSTANT = 'CUSTOMER_360_FEATURE';
+export const CUSTOMER_360_TRANSLATIONS = 'customer360Translations';
+export const CUSTOMER_360_TRANSLATION_CHUNKS_CONFIG =
+  'customer360TranslationChunksConfig';
+
+export const CUSTOMER_360_SCHEMATICS_CONFIG: SchematicConfig = {
+  library: {
+    featureName: CUSTOMER_360_FEATURE_NAME,
+    mainScope: SPARTACUS_ASM,
+    featureScope: SPARTACUS_CUSTOMER_360,
+  },
+  folderName: ASM_FOLDER_NAME,
+  moduleName: CUSTOMER_360_MODULE_NAME,
+  featureModule: {
+    name: CUSTOMER_360_MODULE,
+    importPath: SPARTACUS_CUSTOMER_360,
+  },
+  rootModule: {
+    name: CUSTOMER_360_ROOT_MODULE,
+    importPath: SPARTACUS_CUSTOMER_360_ROOT,
+  },
+  lazyLoadingChunk: {
+    moduleSpecifier: SPARTACUS_CUSTOMER_360_ROOT,
+    namedImports: [CUSTOMER_360_FEATURE_NAME_CONSTANT],
+  },
+  i18n: {
+    resources: CUSTOMER_360_TRANSLATIONS,
+    chunks: CUSTOMER_360_TRANSLATION_CHUNKS_CONFIG,
+    importPath: SPARTACUS_CUSTOMER_360_ASSETS,
+  },
+  styles: {
+    scssFileName: ASM_SCSS_FILE_NAME,
+    importStyle: SPARTACUS_ASM,
+  },
+  dependencyFeatures: [ASM_FEATURE_NAME, STOREFINDER_FEATURE_NAME],
```

Comment:
[MEDIUM QUESTION]
Do you think we should add here also `ORDER_FEATURE_NAME`?

Reply:
Thank you. I will discuss this with Radhep.

Reply:
I have discussed it with Radhep, and we should include 'ORDER_FEATURE_NAME' in the list.  Thanks